docs/source/develop.rst

```diff
@@ -263,10 +263,10 @@ Docstring testing requires ``graphviz`` to be installed. This can be done via::
 Code Formatting
 ~~~~~~~~~~~~~~~
 
-Dask uses several code linters (flake8, black, isort, pyupgrade), which are enforced by
-CI. Developers should run them locally before they submit a PR, through the single
-command ``pre-commit run --all-files``. This makes sure that linter versions and options
-are aligned for all developers.
+Dask uses several code linters (flake8, black, isort, pyupgrade, mypy), which are
```

Comment:
I could be missing something, but it looks like we're not enforcing `mypy` at the moment. Should we drop `mypy` from this list?

Reply:
Ah, you're right. That's from the initial version with the checker. Yes this paragraph should be reverted for now.